BE/src/main/java/com/codesquad/baseballgame/domain/game/dto/BallCountDto.java

```diff
@@ -0,0 +1,14 @@
+package com.codesquad.baseballgame.domain.game.dto;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+
+@Getter @ToString
```

Comment:
애노테이션은 한 줄에 하나를 쓰는 것을 원칙으로 해 주세요.

```suggestion
@Getter
@ToString